### PR TITLE
Added wind speed and direction

### PR DESCRIPTION
**Pull request by captnstarburst**
Display additional information from weatherstack #23 

Added wind speed and wind direction

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -48,14 +48,14 @@ if (!location) {
...
+      (err, { description, temp, feelsLike, tempScale, windSpeed, windDirection } = {}) => {
         if (err) {
           return spinner.fail(err);
         }
         spinner.succeed(chalk.underline(location));
         console.log(
           chalk.cyanBright(
-            `${description}. It is currently ${temp}${tempScale}, it feels like ${feelsLike}${tempScale}.`
+            `${description}. It is currently ${temp}${tempScale}, it feels like ${feelsLike}${tempScale}. The wind is ${windDirection} at ${windSpeed} mph.` 
           )
```
Comment (jmtellez) [inline app.js:59]: Not handling units, check the weatherstack documentation

File: app.js [hunk 2]
```diff
@@ -70,14 +70,14 @@ if (!location) {
...
-      (err, { description, temp, feelsLike, tempScale } = {}) => {
+      (err, { description, temp, feelsLike, tempScale, windSpeed, windDirection } = {}) => {
         if (err) {
           return spinner.fail(err);
         }
         spinner.succeed(chalk.underline(location));
         console.log(
           chalk.cyanBright(
-            `${description}. It is currently ${temp}${tempScale}, it feels like ${feelsLike}${tempScale}.`
+            `${description}. It is currently ${temp}${tempScale}, it feels like ${feelsLike}${tempScale}. The wind is ${windDirection} at ${windSpeed} mph. `
```
Comment (jmtellez) [inline app.js:80]: same as above comment

**Review by jmtellez — CHANGES_REQUESTED** [review]
https://weatherstack.com/documentation#units_parameter
Need to handle units for wind speed. 
Think of a better way to format the results not all not a single line

File: app.js (hunk 1 above)
Comment (captnstarburst) [reply app.js:59]: I will double check the weather stack docs on handling units. Thank you!

File: app.js (hunk 2 above)
Comment (captnstarburst) [reply app.js:80]: Did you have any suggestions? A quick search for line breaks in template literals has left me unsure. Would just an additional console.log be the most elegant solution?

File: app.js (hunk 1 above)
Comment (jmtellez) [reply app.js:59]: we are already handling units, for metric and scientific Wind Speed/Visibility: Kilometers/Hour for Fahrenheit Wind Speed/Visibility: Miles/Hour. I would just do something similar to `getTemperatureScale()` to handle the speed

**Comment by captnstarburst** [comment]
Just wanted to give you an update. I had to get a new computer and I should have that this Thursday, I will be able to make the changes sometime 10/22.

**Comment by captnstarburst** [comment]
I am no longer able to continue working on this, sorry. So I am closing my pull request.

**Comment by jmtellez** [comment]
You can open PR for adding the info, I can work on formatting and units

**Pull request closed without merging**